src/screens/consul/ConsulIndexScreen.js

```diff
@@ -59,7 +59,8 @@ export const ConsulIndexScreen = ({ navigation, route }) => {
   const bookmarkable = route.params?.bookmarkable;
   const extraQuery = route.params?.extraQuery ?? '';
   const query = route.params?.query ?? '';
-  let type = null;
+  const selectedSortingType = sortingType.find((data) => data.selected);
+  const selectedFilterType = filterType.find((data) => data.selected);
```

Comment:
please place them in the corresponding methods. we do not need them globally. it is only needed in the local methods (list data callback for sorting and effect for filtering)

Reply:
7a3a9dacaabe27e7b1291911444f7d2b05906983